en/log/flight_review.md

```diff
@@ -0,0 +1,331 @@
+The **jamming indicator** should be around or below 40. 
+Values around 80 or higher are too high and the setup must be inspected.
+Signal interference is also noticeable as reduced accuracy and lower number of satellites up to the point where no GPS fix is possible.
+
+This is an example without any interference:
+
+![GPS jamming - good plot](../../assets/flight_log_analysis/flight_review/gps_jamming_good.png)
+
+
+## Thrust and Magnetic Field
+
+The *Thrust and Magnetic Field* plot shows the thrust and the norm of the magnetic sensor measurement vector.
+
+The norm should be constant over the whole flight and uncorrelated with the thrust.
+This is a good example where the norm is very close to constant:
+![Thrust and mag close to constant](../../assets/flight_log_analysis/flight_review/thrust_and_mag_good.png)
+
+*If it is correlated*, it means that the current drawn by the motors (or other consumers) is influencing the magnetic field.
+This must be avoided as it leads to incorrect yaw estimation.
+The following plot shows a strong correlation between the thrust and the norm of the magnetometer:
+![Correlated thrust and mag](../../assets/flight_log_analysis/flight_review/thrust_and_mag_correlated.png)
+
+Solutions to this are:
+- Use an external magnetometer (avoid using the internal magnetometer)
+- If using an external magnetometer, move it further away from strong currents (i.e. by using a (longer) GPS mast).
+
+If the norm is uncorrelated but not constant, most likely it is not properly calibrated.
+However it could also be due to external disturbances (for example when flying close to metal constructs).
+
+This example shows that the norm is non-constant, but it does not correlate with the thrust:
+![Uncorrelated thrust and mag](../../assets/flight_log_analysis/flight_review/thrust_and_mag_uncorrelated_problem.png)
+
+
+## Estimator Watchdog
+
+The *Estimator Watchdog* plot shows the health report of the estimator.
+It should be constant zero.
+
+This is what it should look like if there are no problems:
+![Estimator watchdog - good](../../assets/flight_log_analysis/flight_review/estimator_watchdog_good.png)
+
+If one of the flags is non-zero, the estimator detected a problem that needs to be further investigated.
+Most of the time it is an issue with a sensor, for example magnetometer interferences.
+It usually helps to look at the plots of the corresponding sensor.
+<!-- TODO: separate page for estimator issues? -->
+
+Here is an example with magnetometer problems:
+![Estimator watchdog with magnetometer problems](../../assets/flight_log_analysis/flight_review/estimator_watchdog_mag_problem.png)
+
+
+## Sampling Regularity of Sensor Data
+
+The sampling regularity plot provides insights into problems with the logging system and scheduling.
+
+The amount of **logging dropouts** start to increase if the log buffer is too small, the logging rate is too high or a low-quality SD card is used.
+
+> **Note** Occasional dropouts *are* expected.
```

Comment:
Good SD cards don't have them (or at least only in very rare cases), so I'd change it back to 'can be expected'.

Reply:
Because of the context I interpreted  "can be expected" reads as "any card will have some dropouts". I was slightly unsure though, because you could have actually meant to say "Occasional dropouts are acceptable"

I've changed this to (OK?):

> **Note** Occasional dropouts can be expected on medium quality cards.